deployment/ccip/changeset/test_assertions.go

```diff
@@ -449,7 +481,7 @@ func ConfirmExecWithSeqNrs(
 		return nil, fmt.Errorf("no expected sequence numbers provided")
 	}
 
-	timer := time.NewTimer(3 * time.Minute)
+	timer := time.NewTimer(8 * time.Minute)
```

Comment:
We wait for batch of messages so we need higher timeout. Previously it worked well, because we were doing that sequentially